src/main/java/CodeBuilder.java

```diff
@@ -225,13 +223,18 @@ public void perform(@Nonnull Run<?, ?> build, @Nonnull FilePath ws, @Nonnull Lau
             }
         } while(currentBuild.getBuildStatus().equals(StatusType.IN_PROGRESS.toString()));
 
+        this.codeBuildResult.setBuildInformation(currentBuild.getId(), currentBuild.getArn());
+        BuildArtifacts artifacts = currentBuild.getArtifacts();
+
         if(currentBuild.getBuildStatus().equals(StatusType.SUCCEEDED.toString().toUpperCase(Locale.ENGLISH))) {
             action.setJenkinsBuildSucceeds(true);
             this.codeBuildResult.setSuccess();
+            this.codeBuildResult.setArtifactsLocation(artifacts != null ? artifacts.getLocation() : null);
```

Comment:
I'd consolidate this line and the identical one below to 228. If we set the artifacts location regardless of the build status then we should move it outside of this logic.